From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 23 Jun 2023 15:40:00 +0200
Subject: [PATCH] Remove some unnecessary dependencies

- quarkus-jdbc-mssql and quarkus-jdbc-oracle are for proprietary databases,
  we don't support those.

There are still many useless dependencies, but it's not easy to get rid of them.

--- a/quarkus/deployment/pom.xml
+++ b/quarkus/deployment/pom.xml
@@ -140,6 +140,7 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- XXX-Patched
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-mssql-deployment</artifactId>
@@ -160,6 +161,7 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        -->
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-bootstrap-core</artifactId>
--- a/quarkus/runtime/pom.xml
+++ b/quarkus/runtime/pom.xml
@@ -71,6 +71,7 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-mysql</artifactId>
         </dependency>
+        <!-- XXX-Patched
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-mssql</artifactId>
@@ -89,6 +90,7 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        -->
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core</artifactId>
